_maps/map_files/Theseus/TGS_Theseus.dmm

```diff
@@ -7282,12 +7279,9 @@
 	opacity = 0
 	},
 /obj/machinery/door/poddoor/shutters/almayer{
-	density = 0;
 	dir = 4;
-	icon_state = "shutter0";
 	id = "Conference Room Shutters";
-	name = "\improper Privacy Shutters";
-	opacity = 0
+	name = "\improper Privacy Shutters"
```

Comment:
these will start closed instead of open.

Reply:
Fixed.